react-native-cli/index.js

```diff
@@ -257,7 +258,7 @@ function createProject(name, options) {
 }
 
 function getInstallPackage(rnPackage) {
-  var packageToInstall = 'https://github.com/status-im/react-native-linux#react-native-qt'
+  var packageToInstall = 'https://github.com/MaxRis/react-native-qt#bug/repair-run-ubuntu-#117';
```

Comment:
This line should be reverted back before merge. Left this as is if someone will want to test changes locally.